alpha/external-web/app/routes/about-you.js

```diff
@@ -53,6 +53,24 @@ router.post('/about-you', function (request, response) {
       response.status(500).render('error', { message: error.message, error: error })
     }
   })
+
+  // Set statuses for claimant application.
+  var statuses = {
+    applicationStatus: 'PENDING',
+    incomeVerificationStatus: 'PENDING',
+    relationshipVerificationStatus: 'PENDING'
```

Comment:
don't repeat string values, makes multiple places to change, put a var at top of route with 'PENDING'